docs/src/theme/TOC/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import clsx from 'clsx'
+import TOCItems from '@theme/TOCItems'
+import styles from './styles.module.css'
+import { PremiumSupport } from '../components/PremiumSupport/PremiumSupport'
+// Using a custom className
+// This prevents TOCInline/TOCCollapsible getting highlighted by mistake
+const LINK_CLASS_NAME = 'table-of-contents__link toc-highlight'
+const LINK_ACTIVE_CLASS_NAME = 'table-of-contents__link--active'
+export default function TOC({ className, ...props }) {
```

Comment:
Where is this component being imported? I can't see any place in this PR?

Reply:
Hey, I generated this component automatically through swizzling in Docusaurus (https://docusaurus.io/docs/swizzling) and was slightly modified by me. It replaces the default TOC component, so the export is needed for Docusaurus to recognize and use it